### ISSUE TITLE
Please provide a simple example of the code 

### ISSUE THREAD

**Issue by sculyi** [gregkocher/ParticleFilter#1]
Please provide a simple example to use the class ParticleFilter.
Thanks
@wasd12345

**Comment by waudinio27** [comment]
Yes, this is great. Please show an example with a jupyter notebook  and a time series to predict some steps into the future :-D
